tests/db_utils/db_setup.py

```diff
@@ -132,49 +144,46 @@ def get_insertion_order(conn, gt_csv_files):
 
   gt_dir = f'{data_dir}/ground_truth'
   gt_csv_fnames = glob.glob(f'{gt_dir}/*.csv')
-  # Define an asynchronous function to insert data
-  async with engine.begin() as conn:
-    gt_csv_fnames = await conn.run_sync(get_insertion_order, gt_csv_fnames, )
-    # Create tables
-    for csv_fname in gt_csv_fnames:
-      base_fname = os.path.basename(csv_fname)
-      table_name = base_fname.split('.')[0]
-
-      if only_blob_data and not table_name.startswith('blobs') and not table_name.startswith('mapping_'):
-        continue
-      df = pd.read_csv(csv_fname)
-      for column in df.columns:
-        column_info = extract_column_info(table_name, column)
-        df.rename(columns={column: column_info.name}, inplace=True)
 
-      if table_name.startswith('mapping'):
-        _, _, blob_mapping_table_name = table_name_for_rep_and_topk_and_blob_mapping(['blobs_00'])
-        table_name = blob_mapping_table_name
-      # FIXME: in case of mysql, this function doesn't wait, hence throwing integrity error
```

Comment:
"User
pymysql.err.IntegrityError: (1452, 'Cannot add or update a child row: a foreign key constraint fails (`aidb_gt_8000`.`counts03`, CONSTRAINT `counts03_ibfk_1` FOREIGN KEY (`frame`) REFERENCES `blobs_00` (`frame`))')"
I encountered an Integrity Error due to setting autoincrement=True for a primary column. In the MySQL database, this setting caused the value 0 in the 'frame' column to change to 1. After setting autoincrement=False for this column, the error no longer occurred.